addons/website_livechat/tests/test_livechat_basic_flow.py

```diff
@@ -304,34 +307,38 @@ def test_channel_to_store_after_operator_left(self):
         self.assertFalse(
             channel.channel_member_ids.filtered(lambda m: m.partner_id == self.operator.partner_id)
         )
+        expected_result = [
+            {
+                "anonymous_name": f"Visitor #{self.visitor.id}",
+                "channel_type": "livechat",
+                "country_id": False,
+                "create_uid": self.user_public.id,
+                "default_display_mode": False,
+                "fetchChannelInfoState": "fetched",
+                "id": channel.id,
+                "invited_member_ids": [("ADD", [])],
+                "is_editable": False,
+                "is_pinned": True,
+                "last_interest_dt": fields.Datetime.to_string(channel.last_interest_dt),
+                "livechat_end_dt": fields.Datetime.to_string(agent_left_dt),
+                "livechat_operator_id": self.operator.partner_id.id,
+                "member_count": 1,
+                "message_needaction_counter": 0,
+                "message_needaction_counter_bus_id": 0,
+                "name": f"Visitor #{self.visitor.id} El Deboulonnator",
+                "requested_by_operator": False,
+                "rtc_session_ids": [("ADD", [])],
+                "uuid": channel.uuid,
+            }
+        ]
+        if 'ai.agent' in self.env:
+            expected_result[0]['livechat_with_ai_agent'] = False
```

Comment:
We might want to re-introduce `_filter_channels_fields` for this use case, rather than doing it manually in each test: https://github.com/odoo/odoo/pull/220095/files